tests/scripts/all.sh

```diff
@@ -1030,6 +1030,26 @@ component_test_platform_calloc_macro () {
 component_test_make_shared () {
     msg "build/test: make shared" # ~ 40s
     make SHARED=1 all check -j1
+    ldd programs/util/strerror | grep libmbedcrypto
```

Comment:
Wouldn't be better to check this after build, but before running the tests? (Like in the case of cmake.)

Reply:
It only matters if the builds fails or the tests fail. I agree, but it's not worth reworking the PR just for that.